server/test/controllers/FileControllerTest.java

```diff
@@ -58,28 +57,18 @@ public void show_TIManagedApplicant_redirects() {
   public void show_redirects() {
     Applicant applicant = createApplicantWithMockedProfile();
     String fileKey = fakeFileKey(applicant.id, 1L);
-    Request request = fakeRequest().build();
     Result result = controller.show(request, applicant.id, fileKey).toCompletableFuture().join();
     assertThat(result.status()).isEqualTo(SEE_OTHER);
   }
 
-  @Test
-  public void adminShow_invalidProgram_returnsNotFound() {
```

Comment:
Why remove this? The handler is still valid.

Reply:
done

Reply:
added it back